README.md

```diff
@@ -74,6 +74,7 @@ The internals of `regexp2` always operate on `[]rune` so `Index` and `Length` da
 | named back reference `\k'name'` | no | yes |
 | named ascii character class `[[:foo:]]`| yes | no (yes in RE2 compat mode) |
 | conditionals `(?(expr)yes\|no)` | no | yes |
+| PCRE capture group order | no | no (yes in MaintainCaptureOrder mode) |
```

Comment:
We should document the existence of the modifier and the specific behavior of the mode in the readme.